packages/stack/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stackframe/stack",
-  "version": "2.5.37",
+  "version": "2.5.38",
```

Comment:
shouldn't we undo this until we bump the package versions?

Reply:
The problem is that the tests won't pass anymore. Bumping two versions at once shouldn't be a big problem